src/app/utilities/cookies.js

```diff
@@ -10,7 +10,7 @@ export default class cookies {
         return cookiesObject;
     }
 
-    static add(name, value, path, domain) {
+    static add(name, value, path, domain, sameSite, secure) {
```

Comment:
This is the upper limit of parameters for a method signature, consider breaking into a object